projects/libjpeg-turbo/Dockerfile

```diff
@@ -1,4 +1,5 @@
 # Copyright 2016 Google Inc.
+# Copyright 2022 D. R. Commander
```

Comment:
I see them in other projects.

Reply:
(I mean other projects in this repo.) The problem is that, since I had to add a copyright header to the build script in order for it to pass your pre-checks, I could have either assigned the copyright to Google (which, as an independent OSS developer, I have a lot of trouble with) or assigned it truthfully to myself. I don’t see why it matters, since the license is the same either way.